src/edu/csus/ecs/pc2/core/scoring/IScoringAlgorithm.java

```diff
@@ -28,4 +29,20 @@ public interface IScoringAlgorithm {
      * @return An XML descriptor giving standings properties for each team
      */
     String getStandings(IInternalContest theContest, Properties properties, Log log) throws IllegalContestState;
+    
+    /**
+ * Returns an XML description of the current contest standings.
+     * 
+     * @param theContest
+     *            A proxy object referencing the underlying model describing the contest
+     * @param properties
+     *            general and implementation specific settings.
+     * @param log
+     *            a logger, used to add info to the log file and window.
+     * @return An XML descriptor giving standings properties for each team    
```

Comment:
Please delete this extraneous comment (it is a duplicate of the line two lines below...)

Reply:
cleaned up

Reply:
pushed